dts/nxp/nxp_imx/rt/mimxrt1176dvmaa-pinctrl.dtsi

```diff
@@ -6277,104 +6272,116 @@
+		pinmux = <0x40c94028 5 0x0 0 0x40c94068>;
 		pin-snvs;
 	};
-	/omit-if-no-ref/ iomuxc_snvs_gpio_snvs_07_snvs_lp_tamper07: IOMUXC_SNVS_GPIO_SNVS_07_SNVS_LP_TAMPER07 {
-		pinmux = <0x40c94028 0 0x0 0 0x0>;
+	/omit-if-no-ref/ iomuxc_snvs_gpio_snvs_07_dig_snvs_lp_tamper07: IOMUXC_SNVS_GPIO_SNVS_07_DIG_SNVS_LP_TAMPER07 {
+		pinmux = <0x40c94028 0 0x0 0 0x40c94068>;
 		pin-snvs;
 	};
-	/omit-if-no-ref/ iomuxc_snvs_gpio_snvs_08_gpio13_io11: IOMUXC_SNVS_GPIO_SNVS_08_GPIO13_IO11 {
-		pinmux = <0x40c9402c 5 0x0 0 0x0>;
+	/omit-if-no-ref/ iomuxc_snvs_gpio_snvs_08_dig_gpio13_io11: IOMUXC_SNVS_GPIO_SNVS_08_DIG_GPIO13_IO11 {
+		pinmux = <0x40c9402c 5 0x0 0 0x40c9406c>;
 		pin-snvs;
 	};
-	/omit-if-no-ref/ iomuxc_snvs_gpio_snvs_08_snvs_lp_tamper08: IOMUXC_SNVS_GPIO_SNVS_08_SNVS_LP_TAMPER08 {
-		pinmux = <0x40c9402c 0 0x0 0 0x0>;
+	/omit-if-no-ref/ iomuxc_snvs_gpio_snvs_08_dig_snvs_lp_tamper08: IOMUXC_SNVS_GPIO_SNVS_08_DIG_SNVS_LP_TAMPER08 {
+		pinmux = <0x40c9402c 0 0x0 0 0x40c9406c>;
 		pin-snvs;
 	};
-	/omit-if-no-ref/ iomuxc_snvs_gpio_snvs_09_gpio13_io12: IOMUXC_SNVS_GPIO_SNVS_09_GPIO13_IO12 {
-		pinmux = <0x40c94030 5 0x0 0 0x0>;
+	/omit-if-no-ref/ iomuxc_snvs_gpio_snvs_09_dig_gpio13_io12: IOMUXC_SNVS_GPIO_SNVS_09_DIG_GPIO13_IO12 {
+		pinmux = <0x40c94030 5 0x0 0 0x40c94070>;
 		pin-snvs;
 	};
-	/omit-if-no-ref/ iomuxc_snvs_gpio_snvs_09_snvs_lp_tamper09: IOMUXC_SNVS_GPIO_SNVS_09_SNVS_LP_TAMPER09 {
-		pinmux = <0x40c94030 0 0x0 0 0x0>;
+	/omit-if-no-ref/ iomuxc_snvs_gpio_snvs_09_dig_snvs_lp_tamper09: IOMUXC_SNVS_GPIO_SNVS_09_DIG_SNVS_LP_TAMPER09 {
+		pinmux = <0x40c94030 0 0x0 0 0x40c94070>;
 		pin-snvs;
 	};
-	/omit-if-no-ref/ iomuxc_snvs_pmic_on_req_gpio13_io01: IOMUXC_SNVS_PMIC_ON_REQ_GPIO13_IO01 {
-		pinmux = <0x40c94004 5 0x0 0 0x0>;
+	/omit-if-no-ref/ iomuxc_snvs_onoff_dig_src_reset_b: IOMUXC_SNVS_ONOFF_DIG_SRC_RESET_B {
+		pinmux = <0x0 0 0x0 0 0x40c9403c>;
+		pin-pue;
+	};
+	/omit-if-no-ref/ iomuxc_snvs_pmic_on_req_dig_gpio13_io01: IOMUXC_SNVS_PMIC_ON_REQ_DIG_GPIO13_IO01 {
+		pinmux = <0x40c94004 5 0x0 0 0x40c94044>;
 		pin-snvs;
 	};
-	/omit-if-no-ref/ iomuxc_snvs_pmic_on_req_snvs_lp_pmic_on_req: IOMUXC_SNVS_PMIC_ON_REQ_SNVS_LP_PMIC_ON_REQ {
-		pinmux = <0x40c94004 0 0x0 0 0x0>;
+	/omit-if-no-ref/ iomuxc_snvs_pmic_on_req_dig_snvs_lp_pmic_on_req: IOMUXC_SNVS_PMIC_ON_REQ_DIG_SNVS_LP_PMIC_ON_REQ {
+		pinmux = <0x40c94004 0 0x0 0 0x40c94044>;
 		pin-snvs;
 	};
-	/omit-if-no-ref/ iomuxc_snvs_pmic_stby_req_gpio13_io02: IOMUXC_SNVS_PMIC_STBY_REQ_GPIO13_IO02 {
-		pinmux = <0x40c94008 5 0x0 0 0x0>;
+	/omit-if-no-ref/ iomuxc_snvs_pmic_stby_req_dig_gpio13_io02: IOMUXC_SNVS_PMIC_STBY_REQ_DIG_GPIO13_IO02 {
+		pinmux = <0x40c94008 5 0x0 0 0x40c94048>;
 		pin-snvs;
 	};
-	/omit-if-no-ref/ iomuxc_snvs_pmic_stby_req_pgmc_pmic_vstby_req: IOMUXC_SNVS_PMIC_STBY_REQ_PGMC_PMIC_VSTBY_REQ {
-		pinmux = <0x40c94008 0 0x0 0 0x0>;
+	/omit-if-no-ref/ iomuxc_snvs_pmic_stby_req_dig_pgmc_pmic_vstby_req: IOMUXC_SNVS_PMIC_STBY_REQ_DIG_PGMC_PMIC_VSTBY_REQ {
```

Comment:
Slightly concerned about the pin name changes here- this aligns with the source configuration data used to create these headers (and with the reference manual), but changing the naming here will break builds for customers using these pins downstream

Reply:
We should document in the Zephyr release notes.